### DISCUSSION TITLE
Determine & display number of paginated pages

### DISCUSSION THREAD

**Discussion by kvlkvlkvl** [xenocrat/chyrp-lite#271] [Q&A] [answered]
Any help to determining and display the number (##) of paginated pages?

Looking to put this into some footer navigation with prev/next page links.

````
{% if posts.paginated is defined %}
   <li>Page {{ posts.page }} of ##</li>
{% endif %}
```

**Comment by xenocrat** [comment, marked as answer]
Hello there,

You want the total number of pages? Use [`{{ posts.pages }}`](https://github.com/xenocrat/chyrp-lite/blob/master/includes/class/Paginator.php#L34).

**Reply by kvlkvlkvl** [reply]
It's comical that I didn't land on that option with everything I tried. 

Thanks!
